realsense/plugin.cpp

```diff
@@ -9,20 +9,27 @@
 
 using namespace ILLIXR;
 
-static constexpr int IMAGE_WIDTH = 640;
-static constexpr int IMAGE_HEIGHT = 480;
-static constexpr int FPS = 30;
-static constexpr int GYRO_RATE = 400; // 200 or 400
-static constexpr int ACCEL_RATE = 250; // 63 or 250
+static constexpr int IMAGE_WIDTH_D4XX = 640;
+static constexpr int IMAGE_HEIGHT_D4XX = 480;
+static constexpr int FPS_D4XX = 30;
+static constexpr int GYRO_RATE_D4XX = 400; // 200 or 400
+static constexpr int ACCEL_RATE_D4XX = 250; // 63 or 250
 
+static constexpr int IMAGE_WIDTH_T26X = 848;
+static constexpr int IMAGE_HEIGHT_T26X = 800;
 typedef struct {
-	cv::Mat* img0;
-	cv::Mat* img1;
-	cv::Mat* rgb;
-	cv::Mat* depth;
-	int iteration;
-} cam_type;
+    cv::Mat* img0;
+    cv::Mat* img1;
+    cv::Mat* rgb;
+    cv::Mat* depth;
+    int iteration;
+} cam_type_D4XX;
 
+typedef struct {
+    cv::Mat* img0;
+    cv::Mat* img1;
+    int iteration;
+} cam_type_T26X;
```

Comment:
Note that this suggestion has a lot of parallels to what's already going on in [`common/data_format.hpp`](https://github.com/ltstein/ILLIXR/blob/2f89db2595b68e6da53d48ac9db4fc47e7ed3741/common/data_format.hpp#L29), so it could be worth constructing these data types directly without the auxiliary types.